<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelConfigWidget</class>
 <widget class="QWidget" name="ChannelConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Mode</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="OpTopic_">
           <property name="text">
            <string>Only channel  operator can change topic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BlockOutMessage_">
           <property name="text">
            <string>Block messages from outside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="SecretChannel_">
           <property name="text">
            <string>Secret channel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="PrivateChannel_">
           <property name="text">
            <string>Private channel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="InvitesOnly_">
           <property name="text">
            <string>Only by invite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ModerateChannel_">
           <property name="text">
            <string>Moderate channel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ReOp_">
           <property name="text">
            <string>Server Reop Flag</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="UserLimit_">
             <property name="text">
              <string>Max users:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="Limit_"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="Password_">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Key_">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ban list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BanSearch_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Host mask:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BanHostMask_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UpdateBan_">
           <property name="text">
            <string>Update</string>
           </property>
           <property name="icon">
            <iconset theme="document-edit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddBan_">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset theme="list-add"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveBan_">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="BanList_"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Exception list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ExceptSearch_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="ExceptList_"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Host mask:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ExceptHostMask_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UpdateExcept_">
           <property name="text">
            <string>Update</string>
           </property>
           <property name="icon">
            <iconset theme="document-edit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddExcept_">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset theme="list-add"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveExcept_">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Invite list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InviteSearch_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Host mask:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InviteHostMask_">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="UpdateInvite_">
           <property name="text">
            <string>Update</string>
           </property>
           <property name="icon">
            <iconset theme="document-edit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddInvite_">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset theme="list-add"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveInvite_">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="InviteList_"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UserLimit_</sender>
   <signal>clicked(bool)</signal>
   <receiver>Limit_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Password_</sender>
   <signal>clicked(bool)</signal>
   <receiver>Key_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
